src/IssueViz.Security/OpenInIDE/Http/HttpListenerFactory.cs

```diff
@@ -20,14 +20,16 @@
 
 using System;
 using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.Threading.Tasks;
 using Microsoft.Owin.BuilderProperties;
 using Microsoft.Owin.Host.HttpListener;
 using SonarLint.VisualStudio.Core;
 using SonarLint.VisualStudio.Integration;
 
-namespace SonarLint.VisualStudio.IssueVisualization.Security.OpenInIDE
+namespace SonarLint.VisualStudio.IssueVisualization.Security.OpenInIDE.Http
 {
-    internal interface IListenerFactory
+    internal interface IHttpListenerFactory
```

Comment:
I've tried to make the difference between the HTTP and non-HTTP components clearer by some renamings and by creating a folder called HTTP.